tools/bin/ecflow/processes/main/meteorological/reorder.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python
+#####reorder.py
+#####usage: <python> <reorder.py> <configuration.cfg>
+
+#####In order for tonic (netcdf2vic.py) to read and convert these netcdf files
+#####their dimensions must be day,lat,lon.
+#####This script uses subprocess to execute the ncpdq command.
+
+import os
+import sys
+import argparse
+from tonic.io import read_config
+from monitor.io import proc_subprocess
+
+######### ----------------------------------------###########
+
+
+#read in configuration file
+parser = argparse.ArgumentParser(description='Reorder dimensions')
+parser.add_argument('config_file', metavar='config_file', type=argparse.FileType('r'), nargs=1, help='configuration file')
+args = parser.parse_args()
+config_dict = read_config(args.config_file[0].name)
+
+#read in meterological data location
+met_loc = config_dict['ECFLOW']['Met_Loc']
+
+#netcdf file prefixes
+param = ['pr', 'tmmn', 'tmmx', 'vs']
+
+for i in param:
+	#in file
+	nc_file = '%s/%s.nc' %(met_loc, i)
+	#out file
+	reorder_file = '%s/%s.reorder.nc' %(met_loc, i)
+	try:
+		#remove previous days file, ncpdq doesn't overwrite
+		os.remove(reorder_file)
+
+		reorder = ['ncpdq -a day,lat,lon %s %s' %(nc_file, reorder_file)]
+		proc_subprocess(reorder, met_loc)
+	except OSError:
+		print('No file to remove')
+		reorder = ['ncpdq -a day,lat,lon %s %s' %(nc_file, reorder_file)]
+		proc_subprocess(reorder, met_loc)
```

Comment:
rather than this `try/except` statement here to catch the `os.remove` error, why not an if statement that looks something like this:

```Python
if os.path.isfile(reorder_file):
    os.remove(reorder_file)
nco.ncpdq(input=nc_file, output=reorder_file, ...)
```